src/Integration.Vsix/SonarLintIntegrationPackage.cs

```diff
@@ -69,9 +71,12 @@ protected override void Initialize()
             IServiceProvider serviceProvider = this;
 
             var componentModel = serviceProvider.GetService<SComponentModel, IComponentModel>();
-            var activeSolutioNBoundTracker = serviceProvider.GetMefService<IActiveSolutionBoundTracker>();
+            var activeSolutionBoundTracker = serviceProvider.GetMefService<IActiveSolutionBoundTracker>();
             var workspace = componentModel.GetService<VisualStudioWorkspace>();
-            this.sonarAnalyzerManager = new SonarAnalyzerManager(activeSolutioNBoundTracker, workspace);
+            var host = serviceProvider.GetMefService<IHost>();
+            var projectSystemHelper = host.GetService<IProjectSystemHelper>();
```

Comment:
Minor: move this line to where the _ruleSetManager_ is constructed, since that's the only place it's used.